config/crds/v1/all-crds.yaml

```diff
@@ -2833,6 +2833,80 @@ spec:
               image:
                 description: Image is the Elasticsearch Docker image to deploy.
                 type: string
+              monitoring:
+                description: Monitoring enables you to collect and ship log and monitoring
+                  data of this Elasticsearch cluster. See https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html.
+                  Metricbeat and Filebeat are deployed in the same Pod as sidecars
+                  and each one sends data to one or two different Elasticsearch monitoring
+                  clusters running in the same Kubernetes cluster.
+                properties:
+                  logs:
+                    description: Logs holds references to Elasticsearch clusters which
```

Comment:
```suggestion
                    description: Holds references to Elasticsearch clusters which
```

Reply:
It is imposed by Godoc, the first word of the description in comment must correspond to the described property.